src/vm/virtualcallstub.h

```diff
@@ -555,6 +556,10 @@ class VirtualCallStubManager : public StubManager
 #endif                            
                                    );
 
+#ifdef _TARGET_X86_
+    friend void BackPatchWorkerStaticStub(PCODE returnAddr, TADDR siteAddrForRegisterIndirect);
```

Comment:
Similarly as above, it would be better to use ``#if defined(_TARGET_X86_) && defined(FEATURE_PAL)``